package.json

```diff
@@ -29,16 +30,22 @@
   "author": "Improbable",
   "license": "Apache-2.0",
   "dependencies": {
+    "@babel/polyfill": "^7.0.0",
```

Comment:
Shouldn't this be a devDependency?

Reply:
I think I can remove this actually, but no it's the package that has polyfills used at runtime.